src/components/Modal/CreateKeywordModal.jsx

```diff
@@ -70,87 +81,131 @@ const CreateKeywordModal = () => {
 
     setSelectedGroup(inputValue.newGroup);
     setGroupList((prev) => [...prev, newGroup]);
+    setIsCreatingNewGroup(false);
   };
 
   const handleKeywordSubmit = (e) => {
     e.preventDefault();
 
-    if (inputValue.keyword === "") {
+    const keywordValue = inputValue.keyword;
+
+    if (keywordValue === "") {
```

Comment:
```suggestion
    if (keywordValue.trim() === "") {
```
띄어쓰기만 해도 키워드로 등록될 수 있는 상황에 대해서도 처리가 필요할 것 같아요!
trim()을 적용한 값 또한 빈문자열인 경우, 에러 메시지를 보여주면 될 것 같아요

Reply:
좋은 의견 감사합니다. 이 부분을 놓쳤네요,,! 반영하겠습니다 :)

Reply:
수정사항 확인했습니다! 감사합니다 🙇‍♀️